ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/measurement/v2/measurement_service.proto

```diff
@@ -0,0 +1,160 @@
+
+  // Required. Defines the outputs for the measurement.
+  repeated Output outputs = 5;
+}
+
+// Contains measurement User Interface details.
+message UserInterfaceDetails{
+  // Optional. The URL to the file (such as .measui or .vi) providing a user interface for the measurement.
+  string file_url = 1;
+}
+
+// Message that defines a configuration parameter for the measurement.
+message ConfigurationParameter {
+  // Required. The field number for the configuration parameter as defined by the message indicated by
+  // MethodSignature.configuration_parameters_message_type.
+  int32 field_number = 1;
+
+  // Required. The data type for the configuration parameter.
+  google.protobuf.Field.Kind type = 2;
+
+  // Required. The name of the configuration parameter. When defining a user interface for the measurement, a control
+  // that matches this name will be used to supply a value to this configuration parameter.
+  string name = 3;
+
+  // Required. True if this configuration parameter represents repeated data and False if it represents a scalar value.
+  bool repeated = 4;
+
+  // Optional. Represents a set of annotations on the type.
+  // Well-known annotations:
+  // - Type specialization. The keys to other annotations will be read based on the value of `ni/type_specialization` annotation.
+  //   - Key: "ni/type_specialization"
+  //   - Common Values: "pin" ...    
+  // - For string parameter with ni/type_specialization annotation equals "pin"
+  //   - Key: "ni/pin.instrument_type"
+  //   - Common Values: "niDCPower", "niScope"...
+  map<string, string> annotations = 5;
+}
+
+// Message that defines an output of the measurement.
+message Output {
+  // Required. The field number for the output as defined by the message indicated by
+  // MethodSignature.outputs_message_type.
+  int32 field_number = 1;
+
+  // Required. The data type for the output.
+  google.protobuf.Field.Kind type = 2;
+
+  // Required. The name of the output. When defining a user interface for the measurement, an indicator
+  // that matches this name will be used to display the value for this output.
+  string name = 3;
+
+  // Required. True if this output represents repeated data and False if it represents a scalar value.
+  bool repeated = 4;
+
+  // Optional. Represents a set of annotations on the type.
+  // See documentation for ConfigurationParameter annotations for more details and examples.
+  // Note that enum annotations only apply to TYPE_ENUM parameters and not TYPE_STRING parameters.
+  // Well-known annotations:
+  // - Type specialization. The keys to other annotations will be read based on the value of `ni/type_specialization` annotation.
+  //   - Key: "ni/type_specialization"
+  //   - Common Values: "enum" ...    
+  // - For string parameter with ni/type_specialization annotation equals "enum"
+  //   - Key: "ni/enum.values"
+  //   - Expected format: JSON dictionary string"
+  //   - Example: "{\"RED\":0, \"GREEN\":25, \"BLUE\":5}"
```

Comment:
FYI, I posted my comments on this on the corresponding Azure Devops PR (albeit late)

Reply:
[Addressed those comments in ASW](https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/475543) and pulled the changes into this PR.